tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -344,9 +345,106 @@ def test_tag_object_invalid_tag(self) -> None:
                 ["Eukaryota Xenomorph"],
                 "biology101",
             )
-        assert "Invalid object tag for taxonomy (1): Eukaryota Xenomorph" in str(
-            exc.exception
+        assert "Invalid object tag for taxonomy (1): Eukaryota Xenomorph" in str(exc.exception)
+
+    def test_tag_object_string(self):
```

Comment:
If there are no type declarations on the function definition, please add `-> None` so it becomes `def test(self) -> None:` . Specifying any type hints, including just `-> None` will enable type checking for the code in this function. But if there are no type annotations at all in the function declaration then the whole function will be ignored and can contain type errors.

Reply:
Thank you for pointing out! I think we have some type errors here. I will check it out.

Reply:
Done 66e5f2a9c06cc923be11fbcc4dc81dc036df62e3